### ISSUE TITLE
Doc missing

### ISSUE THREAD

**Issue by piotrmilos** [deepsense-ai/mrunner#5]
In Readme.md the link to documentation does not work

More details may be found in documentation

**Comment by pziecina** [comment]
Yes, you are right I'll fix it. BTW - can you generate and publish documentation as mentioned in https://github.com/deepsense-ai/mrunner/blob/develop/docs/contribution.md. It will make documentation available on: https://deepsense-ai.github.io/mrunner

**Comment by pziecina** [comment]
@piotrmilos I think it is now solved with #6
